engine/src/main/java/org/terasology/rendering/opengl/PostProcessor.java

```diff
@@ -317,9 +308,9 @@ private void generateSkyBand(FBO skyBand) {
         }
 
         skyBand.bind();
-        graphicState.setRenderBufferMask(skyBand, true, false, false);
+        setRenderBufferMask(skyBand, true, false, false);
 
-        setViewportTo(skyBand.dimensions());
+        setViewportToSizeOf(skyBand.dimensions());
```

Comment:
In the previous PR we used a version of this method that had in input the FBO, not its dimension. So, this line would read: setViewportToSideOf(skyBand). Can we use that version instead?